SteakBot.Core/Objects/MemeCommand.cs

```diff
@@ -19,5 +20,57 @@ public MemeCommand(MemeResultType resultType, string name, string value, string
             Value = value;
             Description = description;
         }
+
+        #region IEquitable
+
+        public override bool Equals(object other)
+        {
+            var item = other as MemeCommand;
+            return item != null && Equals(item);
+        }
+
+        public bool Equals(MemeCommand other)
+        {
+            if (ReferenceEquals(other, null))
```

Comment:
Add ReferenceEquals for other and this to check if not passing the same pointer. Currently if other is not equal to null values will be compared directly even if the same pointer as this is passed